models/comments.models.js

```diff
@@ -28,3 +28,23 @@ ORDER BY comments.created_at ASC
  })
 })
 }
+
+exports.postComment = (newComment, article_id) => {
+const username = newComment.username
+const comment = newComment.body
+return db.query(`
+INSERT INTO comments(author, body, article_id) 
+VALUES ($1, $2, $3) 
+RETURNING body 
+;`, [username, comment, article_id])
+.then(({rows}) => {
```

Comment:
This is great, but you should be using `RETURNING *` to return the whole comment object and not just the `body`